elements/lisk-dpos/src/rounds.ts

```diff
@@ -13,14 +13,32 @@
  */
 
 interface RoundConstructor {
+	readonly genesisHeight: number;
+	readonly initRound: number;
 	readonly blocksPerRound: number;
 }
 
 export class Rounds {
 	public readonly blocksPerRound: number;
+	public readonly initRound: number;
+	private readonly _genesisHeight: number;
 
-	public constructor({ blocksPerRound }: RoundConstructor) {
+	public constructor({
+		blocksPerRound,
+		genesisHeight,
+		initRound,
+	}: RoundConstructor) {
 		this.blocksPerRound = blocksPerRound;
+		this._genesisHeight = genesisHeight;
+		this.initRound = initRound;
+	}
+
+	public isBootstrapPeriod(height: number): boolean {
+		return height > this._genesisHeight && height <= this.lastHeightBootstrap();
```

Comment:
```suggestion
		return height >= this._genesisHeight && height <= this.lastHeightBootstrap();
```

Reply:
I don't know if this function is every called with the height of a genesis block, but note that the genesis block itself does not belong to a round (it defines an initial state). The first child block of the genesis block is the first block of a round.

Reply:
genesis height will not be the bootstrap period, also this equation was defined in the LIP

Reply:
I also think the current code should work fine.